0-ad-history-mod-package/simulation/templates/units/han/cavalry_spearman_b.xml

```diff
@@ -8,6 +8,12 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        During the early Han period, cavalry was more important than anywhere else in Chinese history. The Xiognu, a nomadic horse people, posed a constant threat and were frequently raiding the dynasty's northern regions. And for the Han, the only way to defend their borders was (besides building a wall, of course) their own cavalry force.          
+        Han cavalrymen were equipped with either a bow, sword, halberd, or lance. Horses and riders wore plate armor made of iron, bronze, leather, or cloth. While this was, in comparison to nomads, heavy protection, Han cavalry units remained highly mobile.         
+        A variety of tactics were applied by the Han cavalry force, many of which they adopted from the Xiognu. The most prevalent were typical melee cavalry charges. But ambushes and hit-and-run weren't uncommon either, as were less conventional tactics like feigning retreats. Cavalry was often supported by infantry and chariots or was used as an additional support force itself.          
```

Comment:
I like this paragraph. It gives players some ideas of what they can do with their calvary.

```suggestion
        A variety of tactics were applied by the Han cavalry force, many of which they adopted from the Xiognu. The most prevalent were typical melee cavalry charges. But ambushes and hit-and-run weren't uncommon either, as were less conventional tactics like feigning retreats. Cavalry was often supported by infantry and chariots, or was used as a support force itself.
```